compiler/luci-interpreter/src/kernels/Utils.h

```diff
@@ -94,6 +94,14 @@ void quantizeMultiplierSmallerThanOneExp(double double_multiplier, int32_t *quan
 
 Shape calculateShapeForBroadcast(const Shape &input1_shape, const Shape &input2_shape);
 
+inline void getQuantizedConvolutionMultipler(float input_scale, float filter_scale,
```

Comment:
Why don't return multiplier by value?

Reply:
When I refer from this function in TFlite, this function need to return 2 value(State code and multiplier). and while make a change for our interpreter, I just removed state return code and make return void. I think this is my mistake. I will fix it.